### PR TITLE
smbmount - improve max_count wording

### PR DESCRIPTION
**Pull request by szaimen**
fix https://help.nextcloud.com/t/smb-mount-max-count/110224
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
This is ready 👍

File: apps/smbmount.sh [hunk 1]
```diff
@@ -77,9 +77,9 @@ then
     msg_box "All $MAX_COUNT slots are occupied. No mounting slots available. Please delete one of the SMB-mounts.
 If you really want to mount more, you can simply download the smb-mount script \
 directly and edit the variable 'MAX_COUNT' to a higher value than $MAX_COUNT by running:
-'curl -sLO https://raw.githubusercontent.com/nextcloud/vm/master/apps/smbmount.sh /var/scripts'
-'sudo nano /var/scripts/smbmount.sh' # Edit MAX_COUNT=$MAX_COUNT to your likings and save the file
-'sudo bash /var/scripts/smbmount.sh' # Execute the script." "$SUBTITLE"
+'wget https://raw.githubusercontent.com/nextcloud/vm/master/apps/smbmount.sh' # Download the script
```
Comment (enoch85) [inline apps/smbmount.sh:80]: please don't use `wget`. We migrated away from that some time ago. 

Use `curl_to_dir` instead or the original `curl` command.

Comment (szaimen) [reply apps/smbmount.sh:80]: Yes, but those are manual commands and don't get executed. Hence I cannot recommend to use curl_to_dir here.
(if you look correctly, this is inside a msg_box !)

Comment (enoch85) [reply apps/smbmount.sh:80]: Yes, that's why I added

> or the original curl command.

;)

Comment (szaimen) [reply apps/smbmount.sh:80]: okay...

**Comment by szaimen** [comment]
Now it is ready 👍

**Pull request merged**